src/main/java/com/google/opensesame/servlets/UserServlet.java

```diff
@@ -55,16 +72,29 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
   @Override
   // Send a user to datastore. Update the current information about the user if one exists.
   public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    String userGitHubID = request.getParameter("githubID");
-    if (userGitHubID.trim().isEmpty()) {
-      response.sendError(HttpServletResponse.SC_BAD_REQUEST, "Invalid ID");
-      return;
-    }
-    ArrayList<String> userTags = new ArrayList<>(Arrays.asList(request.getParameterValues("tags")));
+    try {
+      String userGitHubID = request.getParameter("githubID");
+      if (userGitHubID == null || userGitHubID.trim().length() == 0) {
+        ErrorResponse.sendJsonError(
+            response,
+            "Empty GitHub ID",
+            HttpServletResponse.SC_BAD_REQUEST,
+            "There is no GitHub ID affiliated with this request");
+        return;
+      }
+      ArrayList<String> userTags =
+          new ArrayList<>(Arrays.asList(request.getParameterValues("tags")));
```

Comment:
Does this part need to be in the `try` block? Best to keep the scope of it as limited as possible and move these lines out if not.